### DISCUSSION TITLE
[BUG]:部署cpu不支持vextracti128指令，造成core down

### DISCUSSION THREAD

**Discussion by zliu789** [alibaba/ilogtail#1554] [Help] [answered]
**Describe the bug**
A clear and concise description of what the bug is, ideally within 20 words.

**iLogtail Running Environment**
Please provide the following information:

vextracti128是avx2的，cpu型号不支持avx2指令集，造成启动coredown

(gdb)  display/i $pc
1: x/i $pc
=> 0x1de6222 <_ZN5folly6detail5thunk4makeINS0_15SingletonHolderIN8facebook5velox4tpch12_GLOBAL__N_112DBGenBackendEE4ImplINS0_10DefaultTagESB_EEJEEEPvDpT0_+98>:      vextracti128 $0x1,%ymm0,0x68(%r12)

![微信图片_20240607101247](https://github.com/alibaba/ilogtail/assets/7993101/ad4852b0-88c6-4758-99b5-b1590177d7a0)

- ilogtail version:
  2.0
- Yaml configuration:

- ilogtail.LOG:
无，启动不成功
- logtail_plugin.LOG:
无

**Comment by messixukejia** [comment, marked as answer]
https://github.com/alibaba/ilogtail/discussions/1398

可以看下这个讨论。
